commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_definition_subscription_info.jsp

```diff
@@ -179,16 +171,12 @@ boolean ending = maxSubscriptionCycles > 0;
 				animatingChange: function(event) {
 					var instance = this;
 
-					var expanded = !instance.get('expanded');
+					var expanded = !instance.get('expanded')
```

Comment:
Is there a particular reason you removed the semicolons from these lines? (Here and 2 other files.)